ratelimits/limit.go

```diff
@@ -327,3 +359,103 @@ func (l *limitRegistry) getLimit(name Name, bucketKey string) (*limit, error) {
 	}
 	return nil, errLimitDisabled
 }
+
+// LoadOverridesByBucketKey loads the overrides YAML at the supplied path,
+// parses it with the existing helpers, and returns the resulting limits map
+// keyed by "<name>:<id>". This function is exported to support admin tooling
+// used during the migration from overrides.yaml to the overrides database
+// table.
+func LoadOverridesByBucketKey(path string) (Limits, error) {
+	ovs, err := loadOverrides(path)
+	if err != nil {
+		return nil, err
+	}
+	return parseOverrideLimits(ovs)
+}
+
+// DumpOverrides writes the provided overrides to CSV at the supplied path. Each
+// override is written as a single row, one per ID. Rows are sorted in the
+// following order:
```

Comment:
Nit: I got myself mildly confused on the first read-through that this is talking about the sort order, and not the order of columns in the exported CSV. Mostly my fault, but maybe could be clarified here.

Reply:
I'm a little unclear on this comment, could you elaborate? It is talking about the sort order of the exported CSV.

Reply:
I had misread it as being about both the order of columns and the sort order, so the comment is really just about my own confusion. It might be a good thing to clarify - but OTOH, I can't readily think of an elegant way to clarify it, myself! Feel free to ignore this one.